### ISSUE TITLE
ChannelGroup identifier

### ISSUE THREAD

**Issue by gdenfield** [atlab/sessions#12]
@aecker I've noticed a couple potential issues now with the channel groupings, which I wish I had noticed before processing so much data. They aren't major problems, though, at least not at this time. If we change parameters, they may become an issue.

The primary issue is that there is currently no identifier for which channel grouping parameters were used for detecting and sorting a specific session. That is, if we decided to change the channel count or stride associated with an array_id and began processing more sessions, it would not be obvious which sessions were processed with which set of parameters, other than by the number of electrodes that result. If I've missed where this occurs, please let me know.

Another disadvantage of this grouping approach is that we lose depth resolution, as the channel from which identified single units originated is not saved, only the electrode number determined by the grouping parameters. Thus, we know a range of depths to which that unit might belong but not its precise depth any longer. We may need to figure out how to improve upon this or decide it's not really an issue..

**Comment by aecker** [comment]
Good call. Let's make sure not to change anything about this right now and deal with it when I'm back.

You can get the depth information from the average waveform, which AFAIK is saved for each unit in the `sort.KalmanUnits` table.
